kendo/angular-v-react/article.md

```diff
@@ -449,7 +448,7 @@ It’s also worth establishing conventions within your own organization so your
 
 If you came to this article looking for me to strongly recommend one framework over another, well then I have some bad news for you. Despite the differences between Angular and React, my single biggest finding is that, at the end of the day, the two frameworks are remarkably similar.
 
-React and Angular both do data binding; they both have a rich templating syntax; they both handle events; they both have routers; and they both let you use modern JavaScript features.
```

Comment:
Can you use semicolons like this?

Reply:
I _think_ so? But the commas read cleaner so I like the change.